frontend/src/workspace/globals/globals-themes.js

```diff
@@ -6,7 +6,7 @@
 const workspaceBackground = {
   classic: "#FAFBFF",
   dusk: "#edf2fd",
-  dark: "black",
+  dark: "#525252",
```

Comment:
We personally prefer a darker gray for dark mode. Maybe not black but pretty close. Maybe #161618 ?